src/components/ProgressCircular/readme.md

```diff
@@ -34,9 +33,7 @@ function ActiveUsersCard({ title, variant, color }) {
                 <ProgressCircular variant={variant} value={percent} />
             </div>
             <div className="rainbow-flex rainbow_vertical-stretch">
-                <span style={IconStyles} className={`rainbow-color_${color}`}>
-                    <FontAwesomeIcon icon={faCircle} />
-                </span>
+                <Icon variant={variant} icon={faCircle} />
```

Comment:
Instead of creating a new component for this I think you can try to create a styled component here in the example.